codec_specs.md

```diff
@@ -58,96 +58,159 @@ A timestamp, expressed in [@!RFC3339] that notes when support for the `Codec Map
-
-  
-**Codec ID:** V_REAL/????  
-**Codec Name:** Real Video(TM)   
-**Description:** The stream is one of the Real Video(TM) video streams listed below. Source for the codec names are from [Karl Lillevold on Doom9](http://forum.doom9.org/showthread.php?s=&amp;threadid=55773&amp;perpage=20&amp;pagenumber=2#post331855). The CodecPrivate element contains a "real_video_props_t" structure in Big Endian byte order as found in [librmff](https://github.com/mbunkus/mkvtoolnix/blob/master/lib/librmff/librmff.h).
-
-  
-**Codec ID:** V_REAL/RV10  
-**Codec Name:** RealVideo 1.0 aka RealVideo 5  
-**Description:** Individual slices from the Real container are combined into a single frame.
-
-  
-**Codec ID:** V_REAL/RV20  
-**Codec Name:** RealVideo G2 and RealVideo G2+SVT  
-**Description:** Individual slices from the Real container are combined into a single frame.
-
-  
-**Codec ID:** V_REAL/RV30  
-**Codec Name:** RealVideo 8  
-**Description:** Individual slices from the Real container are combined into a single frame.
-
-  
-**Codec ID:** V_REAL/RV40  
-**Codec Name:** rv40 : RealVideo 9  
-**Description:** Individual slices from the Real container are combined into a single frame.
-
-  
-**Codec ID:** V_QUICKTIME  
-**Codec Name:** Video taken from QuickTime(TM) files  
-**Description:** Several codecs as stored in QuickTime, e.g. Sorenson or Cinepak. The CodecPrivate contains all additional data that is stored in the 'stsd' (sample description) atom in the QuickTime file **after** the mandatory video descriptor structure (starting with the size and FourCC fields). For an explanation of the QuickTime file format read [QuickTime File Format Specification](https://developer.apple.com/library/mac/documentation/QuickTime/QTFF/QTFFPreface/qtffPreface.html).
-
-  
-**Codec ID:** V_THEORA  
-**Codec Name:** Theora  
-**Description:** The private data contains the first three Theora packets in order. The lengths of the packets precedes them. The actual layout is: 
-
-* Byte 1: number of distinct packets '`#p`' minus one inside the CodecPrivate block. This MUST be '2' for current (as of 2016-07-08) Theora headers. 
-* Bytes 2..n: lengths of the first '`#p`' packets, coded in [Xiph-style lacing]({{site.baseurl}}/index.html#lacing). The length of the last packet is the length of the CodecPrivate block minus the lengths coded in these bytes minus one. 
+## Video Codec Mappings
+
+### V_MS/VFW/FOURCC
+
+Codec ID: `V_MS/VFW/FOURCC`
+
+Codec Name: Microsoft (TM) Video Codec Manager (VCM)
+
+Description: The private data contains the VCM structure BITMAPINFOHEADER including the extra private bytes, as [defined by Microsoft](http://msdn.microsoft.com/library/default.asp?url=/library/en-us/gdi/bitmaps_1rw2.asp). The data are stored in little endian format (like on IA32 machines). Where is the Huffman table stored in HuffYUV, not AVISTREAMINFO ??? And the FourCC, not in AVISTREAMINFO.fccHandler ???
```

Comment:
replacing the url with https://msdn.microsoft.com/en-us/library/windows/desktop/dd318229(v=vs.85).aspx which also seems to define BITMAPINFOHEADER